src/EditorFeatures/VisualBasic/LineCommit/CommitFormatter.vb

```diff
@@ -87,6 +90,20 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
             End Using
         End Sub
 
+        Private Function AbortForDiagnostics(document As Document, textSpanToFormat As TextSpan, cancellationToken As CancellationToken) As Boolean
+            Const UnterminatedStringId = "BC30648"
+
+            Dim tree = document.GetSyntaxTreeAsync(cancellationToken).WaitAndGetResult(cancellationToken)
+
+            ' If we have any unterminated strings that overlap what we're trying to format, then
+            ' bail out.  It's quite likely the unterminated string will cause a bunch of code to
```

Comment:
> Rather than bailing the entire format, should we still format the stuff up to (but not including) where the unterminated string literal starts?

I tried that.  But here's the problem.  The unterminated string is generally the last actual string in the document.  But there might be total destruction up until that point.  That's because all previous strings+code have been flipped in the document.